addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15782,32 +15792,32 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36410"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36411"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36412"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36413"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36414"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36415"
-msgid "No info available yet."
+msgid "No information available yet."
```

Comment:
Yes, I wanted to do that but not sure what goes where, that is much much much appreciated, thank you. :)